### ISSUE TITLE
Export for item page -- code (such as JSON) must be keyboard accessible

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#132]
For example, here:
https://stagewebapp1.dlib.nyu.edu/records/mpp7x-hgg82/export/json

From NYU A11y Review: 

In an export page, the section that contains the code cannot be accessed using keyboard-only. Consider adding a "copy to clipboard" option so that the information can be easily accessed.

**Comment by lhenze** [comment]
N/A - this page is removed in future versions
